src/test/regress/expected/multi_name_lengths.out

```diff
@@ -0,0 +1,395 @@
+--
+-- MULTI_NAME_LENGTHS
+--
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 225000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 225000;
+-- Verify that a table name > 56 characters gets hashed properly.
+CREATE TABLE too_long_12345678901234567890123456789012345678901234567890 (
+        col1 integer not null,
+        col2 integer not null);
+SELECT master_create_distributed_table('too_long_12345678901234567890123456789012345678901234567890', 'col1', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('too_long_12345678901234567890123456789012345678901234567890', '2', '2');
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+\c - - - :worker_1_port
+\d too_long_*
+Table "public.too_long_12345678901234567890123456789012345678_e0119164_225000"
+ Column |  Type   | Modifiers 
+--------+---------+-----------
+ col1   | integer | not null
+ col2   | integer | not null
+
+Table "public.too_long_12345678901234567890123456789012345678_e0119164_225001"
+ Column |  Type   | Modifiers 
+--------+---------+-----------
+ col1   | integer | not null
+ col2   | integer | not null
+
+\c - - - :master_port
+-- Verify that the UDF works and rejects bad arguments.
+SELECT shard_name(NULL, 666666);
+ERROR:  object_name cannot be null
+SELECT shard_name(0, 666666);
+ERROR:  object_name does not reference a valid relation
+SELECT shard_name('too_long_12345678901234567890123456789012345678901234567890'::regclass, 666666);
+                           shard_name                            
+-----------------------------------------------------------------
+ too_long_12345678901234567890123456789012345678_e0119164_666666
+(1 row)
+
+SELECT shard_name('too_long_12345678901234567890123456789012345678901234567890'::regclass, NULL);
+ERROR:  shard_id cannot be null
+DROP TABLE too_long_12345678901234567890123456789012345678901234567890 CASCADE;
+-- Table to use for rename checks.
+CREATE TABLE name_lengths (
+	col1 integer not null,
+	col2 integer not null,
+	constraint constraint_a UNIQUE (col1)
+	);
+SELECT master_create_distributed_table('name_lengths', 'col1', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('name_lengths', '2', '2');
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+-- Verify that we CAN add columns with "too-long names", because
+-- the columns' names are not extended in the corresponding shard tables.
+ALTER TABLE name_lengths ADD COLUMN float_col_12345678901234567890123456789012345678901234567890 FLOAT;
+NOTICE:  using one-phase commit for distributed DDL commands
```

Comment:
Might be slightly nicer to put `SET citus.multi_shard_commit_protocol TO '2pc';` at the top of the file.